inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplIntegrationTest.java

```diff
@@ -1538,9 +1537,11 @@ public void readKnowledgeBaseProfiles_ShouldReturnValidHashMapWithProfiles() thr
         String descriptionIri = "http://www.w3.org/2000/01/rdf-schema#comment";
         String propertyLabelIri = "http://www.w3.org/2000/01/rdf-schema#label";
         String propertyDescriptionIri = "http://www.w3.org/2000/01/rdf-schema#comment";
-
-        KnowledgeBaseMapping testMapping = new KnowledgeBaseMapping(classIri, subclassIri, typeIri,
-            descriptionIri, label, propertyTypeIri, propertyLabelIri, propertyDescriptionIri);
+        List<String> rootConcepts =  new ArrayList<>(Arrays.asList(OWL.NOTHING.stringValue()));
```

Comment:
`asList(OWL.NOTHING.stringValue())` (with a static import of `Arrays.asList`) is sufficient unless you need the list to be mutable (which I do assume is not the case).